src/app/downloads/downloads-table-presentation.component.html

```diff
@@ -24,14 +24,14 @@
         </tr>
       </thead>
       <tbody>
-        <tr>
+        <tr *ngIf="podcastTableData">
           <td>
             <prx-checkbox *ngIf="showPodcastToggle; else podcastTitle"
               small [checked]="podcastTableData.charted" [color]="podcastTableData.color"
-              (change)="toggleChartPodcast($event)">{{podcastTableData.title}}</prx-checkbox>
+              (change)="toggleChartPodcast.emit($event)">{{podcastTableData.title}}</prx-checkbox>
             <ng-template #podcastTitle><span class="title">{{podcastTableData.title}}</span></ng-template>
           </td>
-          <td>{{podcastTableData.releaseDate}}</td>
```

Comment:
yep, I just didn't put it on the model for this update. For episodes, I'm formatting the publishedAt field from the template rather than having an already formatted field sitting on the model.